src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -232,32 +232,42 @@ def random_integers(
         and which indexes the returned `pandas.Series`.
     randomness:
         RandomnessStream
+    additional_key:
+        A identifying key for the randomness stream (usually a string)
 
     Returns
     -------
     pandas.Series
         An indexed set of integers in the interval [a,b)
     """
-    return (randomness.get_draw(index=index) * max_val + min_val).round().astype(int)
+    return np.floor(randomness.get_draw(index=index, additional_key=additional_key) * max_val + min_val).astype(int)
 
 
 def filter_by_rate(
-    simulants: pd.Index, randomness: RandomnessStream, rate_producer: Pipeline, additional_key: Any = None
+    sims_to_filter: Union[pd.Index, pd.Series],
+    randomness: RandomnessStream,
+    rate_producer: Pipeline,
+    additional_key: Any = None,
 ) -> pd.Index:
     """
     Parameters
     ----------
-    simulants: a series of every entity that might move. not necessarily unique.
-    move_rate_producer: rate_producer for move rates
+    sims_to_filter: a series of every entity that might move. not necessarily unique.
+    rate_producer: rate_producer for move rates
     randomness: RandomnessStream for component this is being run in
     additional_key: descriptive key to make sure randomness stream produces unique results
 
     Returns
     -------
     a pd.Index, subset from simulants, with those selected to be filtered.
     """
-    simulants = simulants.drop_duplicates()
+    sims_to_filter = sims_to_filter.drop_duplicates()
```

Comment:
sometimes it's not a list of sims, but "household_ids" (or "business_ids") that get passed to this function. in those cases, because there are multiple people per household or business, there can be duplicates.

i think it would make sense to change the signature of this function @albrja so that that is clearer, but maybe not in this PR

Reply:
I think it is in scope to fix the docstring in this PR, since you're updating the functionality.